alpha/action/migrations/migrations.go

```diff
@@ -0,0 +1,122 @@
+package migrations
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+)
+
+type MigrationToken string
+
+const MigrationTokenInvalid MigrationToken = ""
+
+type Migration interface {
+	Token() MigrationToken
+	Name() string
+	Help() string
+	Migrate(*declcfg.DeclarativeConfig) error
+}
+
+func newMigration(name string, help string, fn func(config *declcfg.DeclarativeConfig) error) Migration {
+	s := strings.ReplaceAll(help, "`", "\"")
```

Comment:
This is because of [this](https://github.com/spf13/pflag/issues/200), where expansion of the template for the usage string imparts specific meaning to the first set of backticks it encounters.

Reply:
Instead of doing this "magic" replacement, should we just have the original `help` string use `"` directly?